src/fastoad/module_management/_bundle_loader.py

```diff
@@ -353,33 +376,71 @@ def _install_python_package(self, package_name: str) -> Tuple[Set[Bundle], Set[s
 
         :param package_name:
         :return: A 2-tuple, with the list of installed bundles and the list
-                 of failed modules names
+                of failed modules names
         """
         bundles = set()
         failed = set()
 
         package = PackageReader(package_name)
         if package.has_error or not package.exists:
             failed.add(package_name)
+            _LOGGER.warning(f"Failed to load package: {package_name}")
+            self._styled_rule(f"[bold red]ERROR: {package_name}[/bold red]")
         elif package.is_package:
-            # It is a package, let's explore it.
+            header_printed = False  # to print error header once per package
+
             for item in package.contents:
                 item_package = ".".join([package_name, item])
+
                 if "." in item:
-                    # A file. Considered only if it is a Python file. Ignored otherwise.
+                    # It's a file
```

Comment:
I liked the original comments though I missed them on my first review round ^^' Why delete them ?